### ISSUE TITLE
Istio GKE Deployment Manager Fails

### ISSUE THREAD

**Issue by afroDC** [istio/istio#6598]
**Describe the bug**
Using the provided link for the GKE Istio Deployment manager throws the following error:

```
istio-cluster-2 has resource warnings
istio-cluster: {"ResourceType":"container.v1.cluster","ResourceErrorCode":"400","ResourceErrorMessage":{"code":400,"message":"master version \"1.9.7-gke.1\" is unsupported.","status":"INVALID_ARGUMENT","statusMessage":"Bad Request","requestPath":"https://container.googleapis.com/v1/projects/istio-208316/zones/us-central1-a/clusters","httpMethod":"POST"}}
```

**Expected behavior**

A successful launch. I'm not sure what to expect, I haven't gotten it to work.

**Steps to reproduce the bug**

Click [this link](https://accounts.google.com/signin/v2/identifier?service=cloudconsole&continue=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/release-0.8/install/gcp/deployment_manager/istio-cluster.jinja&followup=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/master/install/gcp/deployment_manager/istio-cluster.jinja&flowName=GlifWebSignIn&flowEntry=ServiceLogin) and try to deploy istio.

**Version**
N/A

**Is Istio Auth enabled or not?**
N/A

**Environment**
GKE

**Comment by ymesika** [comment]
See the discussion in #6520 .
Other devs were using `latest` GKE tag to successfully install it or you can simply use `1.9` to get the lates available 1.9 version from GKE.

We will update the doc and the jinja file to be less coupled with specific versions of GKE platform.

**Comment by afroDC** [comment]
Sounds good. Thanks @ymesika
